PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -89,6 +89,10 @@ def miniAOD_customizeCommon(process):
     #
     process.selectedPatJets.cut = cms.string("pt > 10")
     process.selectedPatMuons.cut = cms.string("pt > 5 || isPFMuon || (pt > 3 && (isGlobalMuon || isStandAloneMuon || numberOfMatches > 0 || muonID('RPCMuLoose')))")
+    
+    from Configuration.Eras.Modifier_phase2_muon_cff import phase2_muon
+    phase2_muon.toModify(process.selectedPatMuons, cut = "pt > 5 || isPFMuon || (pt > 3 && (isGlobalMuon || isStandAloneMuon || numberOfMatches > 0 || muonID('RPCMuLoose') || muonID('ME0MuonArbitrated')))")
```

Comment:
"I am not sure if it worth to add it"
sounds to me partially that it was not included intentionally. ;)

I think that one match in GE2/1 should be clean enough to be kept.
I'm not so sure about just GE1/1: it's probably a yes, considering that the ".OR." is with "numberOfMatches>0".